Clarify configurable back pressure properties (#1515)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-shadow/rpk-shadow-create.adoc

> Before you create a shadow link, generate a configuration file with xref:reference:rpk/rpk-shadow/rpk-shadow-config-generate.adoc[`rpk shadow config generate`] and update it with your source cluster details. The command prompts you to confirm the creation. Use the `--no-confirm` flag to skip the confirmation prompt.

Reviewer: ```suggestion
Before you create a shadow link, generate a configuration file with xref:reference:rpk/rpk-shadow/rpk-shadow-config-generate.adoc[`rpk shadow config generate`] and update it with your source cluster details. For Redpanda Cloud, use the `--for-cloud` flag. The command prompts you to confirm the creation. Use the `--no-confirm` flag to skip the confirmation prompt.
```

> When creating a Shadow Link for Redpanda Cloud, make sure to login and select the cluster where you want to create the Shadow Link before running this command. See xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[`rpk cloud login`] and xref:reference:rpk/rpk-cloud/rpk-cloud-select.adoc[`rpk cloud select`]. For SCRAM authentication, store your password in the secrets store. See xref:reference:rpk/rpk-security/rpk-security-secret.adoc[`rpk security secret --help`] for more details.

Reviewer: ```suggestion
For Redpanda Cloud, first log in and select the cluster where you want to create the shadow link before running this command. See xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[`rpk cloud login`] and xref:reference:rpk/rpk-cloud/rpk-cloud-select.adoc[`rpk cloud select`]. For SCRAM authentication, store your password in the secrets store. See xref:reference:rpk/rpk-security/rpk-security-secret.adoc[`rpk security secret --help`].
```
